tgui/packages/tgui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "name": "tgui",
+  "version": "3.1.0",
```

Comment:
May have misunderstood, but documentation implied this was v4.something?

Reply:
4.1x probably